apps/squeaknode/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+services:
+  web:
+    image: yzernik/squeaknode:v0.1.89@sha256:249337c965855df5bd883a680ff3172993e2273415e63671e31ed4df6850ae5a
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8994:8994
+      - 12994:12994
+      - 8555:8555
+      - 18555:18555
```

Comment:
Are you able to explain (or link me to some docs) regarding what each port is used for?

Reply:
8994 is used by the gRPC server.
12994 is used by the web server.
8555 is used by the p2p protocol in mainnet.
18555 is used by the p2p protocol in testnet.

Reply:
I removed ports 8555 and 18555, since they will be accessible through the tor configuration.